packages/item-cc/src/item.controller.ts

```diff
@@ -196,6 +257,20 @@ export class ItemController extends ConvectorController {
     });
   }
 
+  @Invokable()
+  public async getTransportItems(
+    @Param(yup.string())
+      transporterId: string
+  ) {
+    await ItemController.checkValidOwner(this.sender, transporterId);
```

Comment:
Why can only the transporter himself see the items?

Reply:
The idea is that with this function the transporter receives all items that he needs to transport, and only he can see exactly which items these are. Similarly to getParticipantProposals, where only the proposedOwner can see items that are proposed to him. 

The owner and the proposedOwner can still also see the item though.